examples/log-to-file.md

```diff
@@ -0,0 +1,72 @@
+# Log to File
+
+- [Basic Example](#basic-example)
+- [Separators in JSON Streams](#Separators-in-JSON-streams)
+- [Log Rotation](#log-rotation)
+
+## Basic Example
+
+```
+file: [{
+    module: 'good-squeeze',
+    name: 'Squeeze',
+    args: [{ ops: '*' }]
+}, {
+    module: 'good-squeeze',
+    name: 'SafeJson'
+}, {
+    module: 'good-file',
+    args: ['./test/fixtures/awesome_log']
+}]
+```
+
+This reporter spec logs ops events to a file.
+
+It uses the `good-squeeze` Squeeze function to select only ops events.
+
+Events that pass the filter stream to `good-squeeze` SafeJson for formatting.
```

Comment:
Not really formatting... more like turing into JSON string values. That's important because good-file expects string values and not objects.